tests/apollo/util/skvbc_history_tracker.py

```diff
@@ -949,7 +949,7 @@ async def read_and_track_known_kv(self, key, client):
 
     async def tracked_prime_for_state_transfer(
             self, stale_nodes,
-            checkpoints_num=2,
+            checkpoints_num=20,
```

Comment:
Why change this to 20?

Reply:
Well, essentially, to make the state transfer test non-trivial. With a window of 30, we would generate a checkpoint for every 15 sequence numbers. So with 2 checkpoints, we would only have 30 blocks to transfer, which happens too fast for our test suites to be able to properly detect and inject failures.